wgsl/index.bs

```diff
@@ -567,11 +539,15 @@ global_constant_decl
 variable_decoration_list
   : ATTR_LEFT (variable_decoration COMMA)* variable_decoration ATTR_RIGHT
 
+literal_or_ident
+  : FLOAT_LITERAL
+  | INT_LITERAL
+  | UINT_LITERAL
+  | IDENT
+  | STRING_LITERAL
+
 variable_decoration
-  : LOCATION INT_LITERAL
-  | BUILTIN builtin_decoration
-  | BINDING INT_LITERAL
-  | SET INT_LITERAL
```

Comment:
The possible values for each decoration isn't defined anywhere anymore.

Reply:
Oops, listed now.

Reply:
Can we not mix builtins and intrinsics together in this PR?
I don't see them being the same. The case about intrinsic functions is somewhat strong, since they are, well, functions. The case about builtins is not yet convincing to me and needs to be made first.